### DISCUSSION TITLE
2.8.3

### DISCUSSION THREAD

**Discussion by arianneorpilla** [arianneorpilla/jidoujisho#256] [Announcements]
This is a hotfix release for  [**2.8**](https://github.com/lrorpilla/jidoujisho/releases/tag/2.8.0). Please refer to the [**releases page**](https://github.com/lrorpilla/jidoujisho/releases) for a full timeline of changes. 

**System Requirements**: This application has been tested to work on a variety of devices and emulators running Android 7 to 13. Please note that a significant amount of internal storage space may be taken up by the dictionary database for indexing in order to optimize query speed and performance. Try to give the application around 2-4GB of storage space. Make sure your Android WebView is updated before use. This will resolve common issues that may occur with ッツ Ebook Reader or Mokuro.

# 2.8.3
- Added an option to change the double tap seek duration under the settings for the Player tab.
- Changed the default double tap seek duration to 5 seconds.

# 📲 Which file should I download?
As a rule of thumb, `arm64` should work for most modern mobile devices. `armeabi` should work on older devices. Generally, desktops and emulators should use `x86_64`.

Depending on which device you are using, download the appropriate `APK` by expanding the `Assets` drop-down below and install on your mobile phone by launching the `APK` with your file explorer.

<hr /><em>This discussion was created from the release <a href='https://github.com/lrorpilla/jidoujisho/releases/tag/2.8.3'>2.8.3</a>.</em>

**Comment by brishtibheja** [comment]
Hey, thanks for your amazing work. There is a small request I want to make. With the new browser feature could you also make this available offline such that we could download the webpages to read later. I am wanting to read a web novel recently so such a feature would be great as I get internet connection for only a little time. Thanks, again.

**Reply by arianneorpilla** [reply]
It should be possible, I'll look into it when I have the time.

**Reply by arianneorpilla** [reply]
I tried this tonight but it won't make it tonight's release as it's quite troublesome. Archiving and viewing the site is easy, but injecting JavaScript does not seem to work because of CORS.

**Comment by YoriJaski** [comment]
Why did you remove Text Segmentation for non sentence fields? Is it because of cloze support? It makes using a mono dictionary quite hard if there's an unknown word or few in the definition that I can not look up, I would appreciate it if you would bring back Text Segmentation in non sentence fields. Either way, thank you for the update.

**Reply by arianneorpilla** [reply]
The change was done due to cloze support. I will look into adding it back if possible, but in the meantime users can highlight words to search in those fields.

**Reply by arianneorpilla** [reply]
My advice: view the definition as normal when it is displayed during the search just before you mine a word, and stash those words for use in the Card Creator.

In general, MeCab-based text segmentation is inferior compared to where the user selects the start character, for determining the start and end of words, which is why I have opted to remove it -- but I will see if it's not too hard to bring back.

**Reply by arianneorpilla** [reply]
@YoriJaski I re-released 2.8.4, so your comment was lost when I re-uploaded the release. Hope having the feature back is of help.

**Reply by YoriJaski** [reply]
It is of help, thank you.
